SQL/Archive/2018-03-01_normalise_consent.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'useConsent', 'Enable if the study uses the loris architecture for consent', 1, 0, 'boolean', ID, 'Use consent', 15 FROM ConfigSettings WHERE Name='study';
+INSERT INTO Config (ConfigID, Value) SELECT ID, 'false' FROM ConfigSettings WHERE Name='useConsent';
+
+CREATE TABLE `consent` (
+  `ConsentID` int(2) NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `Label` varchar(255) NOT NULL,
```

Comment:
It seems a bit redundant to have both a name and a label in the consent table, what is the reasoning for this? Since the ConsentID will work as an index, it seems to me that there is only need for one of the two to describe the consent.

Reply:
its for front end use to label e.g. the form elements for that consent. a standard has also just been recently agreed on to have 'Name' and 'Label' for things @ridz1208

Reply:
Do we need 255 chars as maximum for these 2 attributes?